gym/wrappers/test_record_episode_statistics.py

```diff
@@ -22,3 +22,17 @@ def test_record_episode_statistics(env_id, deque_size):
                 break
     assert len(env.return_queue) == deque_size
     assert len(env.length_queue) == deque_size
+
+
+@pytest.mark.parametrize("env_id", ["CartPole-v0"])
+def test_record_episode_statistics_with_vectorenv(env_id):
+    envs = gym.vector.make(env_id, asynchronous=False)
```

Comment:
With the corresponding imports
```python
from gym.vector import AsyncVectorEnv, SyncVectorEnv
from gym.vector.tests.utils import make_env
```

```suggestion
@pytest.mark.parametrize("klass", [SyncVectorEnv, AsyncVectorEnv])
@pytest.mark.parametrize("num_envs", [1, 4])
def test_record_episode_statistics_with_vectorenv(klass, num_envs):
    env_fns = [make_env("CartPole-v0", i) for i in range(num_envs)]
    envs = klass(env_fns)
```

Reply:
Unfortunately it’s gonna fail with `AsyncVectorEnv` because the `envs.env.envs[0].spec.max_episode_steps` is inaccessible. Maybe I should just hardcore a 201 instead of `envs.env.envs[0].spec.max_episode_steps`?

Reply:
Oh sorry I didn't see that you were looping over that later. Then you can ignore this (maybe keeping the parametrization for `num_envs`?).

Reply:
Done. Would you mind Allowing the GitHub action workflow runs? I have some weird setup That makes it difficult to run test cases locally….

Reply:
@jkterry1